zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkConnection.java

```diff
@@ -192,4 +234,60 @@ public String getServers() {
   public void addAuthInfo(String scheme, byte[] auth) {
     _zk.addAuthInfo(scheme, auth);
   }
+
+  private void lookupGetChildrenMethod() {
```

Comment:
I am not sure if I fully understand your point. I guess your point is to reduce the places where `_getChildrenMethod` is referred. I just tried and realized that the code is not as clean as current way, because `lookupNonPaginatedGetChildren` is called twice. It won't achieve "the _getChildrenMethod is only referred in this method".

Since it is an internal variable/cache, I think it's fine to refer it multiple places as current way.

Reply:
Synced with @jiajunwang offline. The style change is not a must. Also updated the style to reduce the reference places.